src/Components/AdvancedSearch/Internal/AdvancedSearchResultDetailsList/AdvancedSearchResultDetailsList.tsx

```diff
@@ -201,11 +199,37 @@ const AdvancedSearchResultDetailsList: React.FC<IAdvancedSearchResultDetailsList
         }
     };
 
+    const getAvailableProperties = () => {
+        let properties: string[] = [];
+        twins.forEach((twin) => {
+            properties = Object.keys(twin).concat(properties);
+        });
+        properties.sort((a, b) => (a.toLowerCase() > b.toLowerCase() ? 1 : -1));
+        return new Set(properties);
+    };
+    const updateColumns = (columnToAdd: string) => {
+        setSelectedColumnNames((currentValue) => {
+            return currentValue.concat(columnToAdd);
+        });
+    };
+    const deleteColumn = (columnToRemoveKey: string) => {
+        setSelectedColumnNames((currentValue) =>
+            currentValue.filter((col) => col !== columnToRemoveKey)
+        );
+    };
     return (
         <section className={classNames.root}>
-            <h3 className={classNames.listHeader}>
-                {t('advancedSearch.results', { twinCount })}
-            </h3>
+            <div className={classNames.listHeaderAndIcon}>
+                <h3 className={classNames.listHeader}>
+                    {t('advancedSearch.results', { twinCount })}
+                </h3>
+                <ColumnPicker
```

Comment:
please hide the picker when there are no results
![image](https://user-images.githubusercontent.com/57726991/193125456-68f76ce4-87d7-408c-9d38-002e6d9d0226.png)